tensorboard/webapp/runs/store/runs_selectors.ts

```diff
@@ -137,7 +142,19 @@ export const getRunGroupBy = createSelector(
   getRunUserSetGroupBy,
   getDataState,
   (userSetGroupBy: GroupBy | null, dataState: RunsDataState): GroupBy => {
-    return userSetGroupBy ?? dataState.initialGroupBy;
+    const regexString = dataState.colorGroupRegexString;
+    if (
+      userSetGroupBy &&
+      userSetGroupBy.hasOwnProperty('key') &&
```

Comment:
Should avoid using `hasOwnProperty` as it will not do the proper thing when JSCompiler mangles property name. Please revert back to the original types and it will be fine.